### PR TITLE
Document FP from #3803

### PR DESCRIPTION
**Pull request by costin-zaharia-sonarsource**
Adds test case to document the FP from #3803

### CODE REVIEW

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM

File: analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MethodParameterUnused.CSharp8.cs [hunk 1]
```diff
@@ -66,6 +68,13 @@ public void Method()
                 var result = a + c;
             }
         }
+
+        // See: https://github.com/SonarSource/sonar-dotnet/issues/3803
+        private void AddValue(uint id1, uint id2, string value) // Noncompliant - FP: id1 and id2 are used inside the tuple
```
Comment (pavel-mikula-sonarsource) [inline analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MethodParameterUnused.CSharp8.cs:73]: CI failed, we'll need one more `//Noncompliant@-1` marker

Comment (costin-zaharia-sonarsource) [reply analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MethodParameterUnused.CSharp8.cs:73]: Indeed, there are two of them 😄

**Review by csaba-sagi-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**